service/gcs/bridge/connection.go

```diff
@@ -5,48 +5,18 @@ import (
 
 	"github.com/pkg/errors"
 
+	"io"
```

Comment:
Oh, good advice! I have vim set to use gofmt, but I can change it to use goimports, and we can slowly fix the import formatting in each file as we go.